src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,13 @@
-const humanCatDogYears = number => {};
-
+const humanCatDogYears = number => {
+    if (number === 1){
+        return [1,15,15];
+    }
+    if (number === 2) {
+        return [2, 24, 24];
+    } else {
```

Comment:
If you have two if statements that return and then you want to do a default case, you don't actually need an else block here as it will naturally run anything after the if statements if both the conditions fail.